packages/tools/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
-	"extends": "../../tsconfig.base.server.json",
+	"extends": "@rocket.chat/tsconfig/server.json",
 	"compilerOptions": {
 		"declaration": true,
 		"rootDir": "./src",
 		"outDir": "./dist"
 	},
-	"include": ["./src/**/*"]
+	"include": ["./src/**/*"],
+	"exclude": ["./src/**/*.spec.ts"]
```

Comment:
why are you adding this exclusion rule to this specific package only?

Reply:
What I meant was: why are the unit tests of this specific package excluded, while other packages still include their own tests?

Reply:
They don't include their own tests, if they did the tests would be transpiled and `*.spec.js`, `*.spec.d.ts` files would be created.

When the tsconfig files were in the root, extending them was excluding all `*.spec.ts` files.

Some packages already were doing this instead of inheriting from base:
https://github.com/RocketChat/Rocket.Chat/pull/35554/files#diff-4ee07dcf11b8207c9823d05ac6ec91bcf152377cadeb557c4d3ac107eb521f78

Reply:
They are transpiling to *.spec.js! See the models package for example.
I'd say to either add the `exclude` line to every package in this PR, or don't add it to any here and do it in a separate PR instead.

Reply:
Got it. I went with the `${configDir}` route since I guess it preserves current behavior.